website/content/docs/connect/observability/access-logs.mdx

```diff
@@ -0,0 +1,255 @@
+
+To use a custom access log, in the `proxy-defaults` configuration entry, set [`JSONFormat`](/docs/connect/config-entries/proxy-defaults#jsonformat) to the string representation of the desired JSON.
+
+Nesting is supported.
+
+<CodeTabs tabs={[ "HCL", "Kubernetes YAML", "JSON" ]}>
+
+```hcl
+Kind      = "proxy-defaults"
+Name      = "global"
+AccessLogs {
+  Enabled = true
+  JSONFormat = <<EOF
+{
+    "myCustomKey" : {
+        "myStartTime" : "%START_TIME%",
+        "myProtocol" : "%PROTOCOL%"
+    }
+}
+EOF
+}
+```
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ProxyDefaults
+metadata:
+  name: global
+spec:
+  accessLogs:
+    enabled: true
+    jsonFormat: |
+      {
+        "myCustomKey" : {
+          "myStartTime" : "%START_TIME%",
+          "myProtocol" : "%PROTOCOL%"
+        }
+      }
+```
+
+```json
+{
+  "Kind": "proxy-defaults",
+  "Name": "global",
+  "AccessLogs": {
+    "Enabled": true,
+    "JSONFormat": "{ \"myCustomKey\" : { \"myStartTime\" : \"%START_TIME%\", \"myProtocol\" : \"%PROTOCOL%\"} }"
+  }
+}
+```
+
+</CodeTabs>
+
+### Text format
+
+To use a custom access log formatted in plaintext, in the `proxy-defaults` configuration entry, set [`TextFormat`](/docs/connect/config-entries/proxy-defaults#textformat) to the desired customized string.
+
+New lines are automatically added to the end of the log to keep each access log on its own line in the output.
+
+<CodeTabs tabs={[ "HCL", "Kubernetes YAML", "JSON" ]}>
+
+```hcl
+Kind      = "proxy-defaults"
+Name      = "global"
+AccessLogs {
+  Enabled = true
+  TextFormat = "MY START TIME: %START_TIME%, THIS CONNECTIONS PROTOCOL IS %PROTOCOL%" 
+}
+```
+
+```yaml
+apiVersion: consul.hashicorp.com/v1alpha1
+kind: ProxyDefaults
+metadata:
+  name: global
+spec:
+  accessLogs:
+    enabled: true
+    textFormat: "MY START TIME: %START_TIME%, THIS CONNECTIONS PROTOCOL IS %PROTOCOL%" 
+```
+
+```json
+{
+  "Kind": "proxy-defaults",
+  "Name": "global",
+  "AccessLogs": {
+    "Enabled": true,
+    "JSONFormat": "MY START TIME: %START_TIME%, THIS CONNECTIONS PROTOCOL IS %PROTOCOL%" 
+  }
+}
+```
+
+</CodeTabs>
+
+
+## Kubernetes
+
+As part of its normal operation, the Envoy debugging logs for the `consul-dataplane`, `envoy`, or `envoy-sidecar` containers are written to `stderr`.
+The access log [`Type`](/docs/connect/config-entries/proxy-defaults#type) is set to `stdout` by default for access logs when enabled.
+Use a log aggregating solution to separate the machine-readable access logs from the Envoy process debug logs.
+
+## Write to a file
+
+You can write access logs to a file on the host where Envoy runs.
+The is most relevant when running Envoy on a VM.
+
+~> **operations warning:** Envoy does not rotate log files. 
+Have a log rotation solution, such as [logrotate](https://www.redhat.com/sysadmin/setting-logrotate), in place to prevent the host's disk space from being consumed by access logs when writing to a file. 
```

Comment:
Linking to a random red hat article might be weird, but since this is a low-level linux tool I didn't have a better suggestion. I'd rather the reader leave with some actionable information.